### PR TITLE
fixes minigui/fetch-and-run.sh

### LINKED ISSUE #681: Help: minigui can't work because "can't open file 'main.py'"
Hi, sir, 
Thank you first, your great effort let me see the amazing AI go.

I wish to run minigo with minigui on google cloud.
I can run minigo with below command, it work well with the gtp command, just like 'genmove b'
```
cd $HOME/github/minigo/
READOUTS=2
LATEST_MODEL=~/minigo-models/models/000990-cormorant
BOARD_SIZE=19 python3 gtp.py --load_file=$LATEST_MODEL --num_readouts=$READOUTS --verbose=3
```

So I try to run ```./minigui/fetch-and-run.sh``` , but failed.
```
Using: the following options for Minigui launch:
--------------------------------------------------
MINIGUI_PYTHON:       python3
MINIGUI_BUCKET_NAME:  minigo-pub
MINIGUI_GCS_DIR:      v15-19x19/models
MINIGUI_MODEL:        000990-cormorant
MINIGUI_MODEL_TMPDIR: /tmp/minigo-models
MINIGUI_BOARD_SIZE:   19
MINIGUI_PORT:         5001
MINIGUI_HOST:         127.0.0.1
Python Version:       Python 3.6.7

Downloading Model files:
--------------------------------------------------
Checking for: /tmp/minigo-models/000990-cormorant.data-00000-of-00001
Checking for: /tmp/minigo-models/000990-cormorant.index
Checking for: /tmp/minigo-models/000990-cormorant.meta

Freezing model
--------------------------------------------------
python3: can't open file 'main.py': [Errno 2] No such file or directory
```

Can you help me to figure it out?

Thank you.

### PR DESCRIPTION
**Pull request by tommadams**
Fixes #681 

Note that Minigui's study mode still won't work after this change. The Python engine doesn't currently support the required GTP command extensions. I'll file a separate issue about that.

### CODE REVIEW

File: minigui/minigui-common.sh [hunk 1]
```diff
@@ -16,9 +16,11 @@
 
 export MINIGUI_PYTHON=${MINIGUI_PYTHON:-"python3"}
 export MINIGUI_BUCKET_NAME=${MINIGUI_BUCKET_NAME:-"minigo-pub"}
-export MINIGUI_GCS_DIR=${MINIGUI_GCS_DIR:-"v7-19x19/models"}
```
Comment (sethtroisi) [inline minigui/minigui-common.sh]: this is pretty old :)

File: minigui/fetch-and-run.sh [hunk 2]
```diff
@@ -48,41 +50,56 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
-    file_to_check="${MINIGUI_MODEL_TMPDIR}/${MINIGUI_MODEL}.${suffix}"
+    file_to_check="${model_tmpdir}/${MINIGUI_MODEL}.${suffix}"
     echo "Checking for: ${file_to_check}"
     if [[ ! -f "${file_to_check}" ]]; then
-      gsutil cp gs://${MINIGUI_BUCKET_NAME}/${MINIGUI_GCS_DIR}/${MINIGUI_MODEL}.${suffix} $MINIGUI_MODEL_TMPDIR/
+      gsutil cp gs://${MINIGUI_BUCKET_NAME}/${MINIGUI_GCS_DIR}/${MINIGUI_MODEL}.${suffix} ${model_tmpdir}/
     fi
   done
 
   # Assume models need to be frozen if .pb doesn't exist.
```
Comment (amj) [inline minigui/fetch-and-run.sh]: if we use anything past v10, the .pb's exist in minigo-pub already

Comment (amj) [reply minigui/fetch-and-run.sh]: as a bonus, that means we don't have to guess at the conv-width param (likely source of bugs)

Comment (tommadams) [reply minigui/fetch-and-run.sh]: We still need conv-width to run inference though

**Pull request merged**